controller/node_controller.go

```diff
@@ -1014,7 +1014,7 @@ func (nc *NodeController) syncInstanceManagers(node *longhorn.Node) error {
 				}
 			}
 			if cleanupRequired {
-				log.Infof("Cleaning up the redundant instance manager %v when there is no running/starting instance", im.Name)
+				log.Infof("Cleaning up the redundant instance manager %v when there is no running/starting instance: %v", im.Name, dumpInstanceState(im.Status.Instances))
```

Comment:
What's the purpose of dumping all instances' state here? Are you looking to determine if any running or starting instances are still present in rare cases?

This could generate a numerous log if there are many instances inside, although this should rarely happen, only when we have multiple IM pods